### PR TITLE
Angular UI webrtc into angular-ui

### PR DESCRIPTION
**Pull request by Blacklotus89898**

### CODE REVIEW

File: .github/workflows/sanity-check.yml [hunk 1]
```diff
@@ -15,13 +15,7 @@ jobs:
     name: Catkin Build Test
     steps:
       - name: Clean Root Workspace
-        uses: AutoModality/action-clean@v1
-        
-      - name: Install Cuda Toolkit
-        uses: Jimver/cuda-toolkit@v0.2.11
-        id: cuda-toolkit
-        with:
-          cuda: '12.1.0'
```
Comment (JosephSaliba01) [inline .github/workflows/sanity-check.yml:24]: Why are we removing this?

File: .github/workflows/sanity-check.yml [hunk 2]
```diff
@@ -54,18 +48,12 @@ jobs:
           cd ~
           mkdir --parents catkin_ws/src
           ln -s $GITHUB_WORKSPACE ~/catkin_ws/src
-
-      - name: Install Intel Realsense SDK
-        run: |
-            sudo apt-get install ros-noetic-ddynamic-reconfigure
-            cd .github/SETUP_SCRIPTS
-            ./installLibrealsense.sh
-            ./buildLibrealsense.sh
```
Comment (JosephSaliba01) [inline .github/workflows/sanity-check.yml:63]: And this

**Review by JosephSaliba01 — CHANGES_REQUESTED** [review]
Good other than my question

File: .github/workflows/sanity-check.yml (hunk 1 above)
Comment (Blacklotus89898) [reply .github/workflows/sanity-check.yml:24]: I honestly do not why it is being removed. I'm might have overwritten it when merging angular-ui into it before the PR, but I will check how it got changed.

Comment (Blacklotus89898) [reply .github/workflows/sanity-check.yml:24]: It was changed when we merged remote tracking into it: b5ab57a771555eeebe8e42227fd7c7895e464aa6
https://github.com/mcgill-robotics/rover/pull/156/commits/0eda0cbd1743e6354d950a5d8cab1e8936586aa1

**Comment by JosephSaliba01** [comment]
@Blacklotus89898 I see that you reverted the changes on the workflow file, you sure these changes shouldn't be there? Is it good now?

**Comment by Blacklotus89898** [comment]
> @Blacklotus89898 I see that you reverted the changes on the workflow file, you sure these changes shouldn't be there? Is it good now?

I think I should double check with Adam just to be sure regarding that file.

**Review by amotaouakkil123 — APPROVED** [review]

**Review by JosephSaliba01 — APPROVED** [review]

**Pull request merged**